### PR TITLE
Readd "Adds Senior Engineering Gear to Atmos loadouts #199"

### PR DESCRIPTION
**Pull request by BeeRobynn**
Check #199 for reference

### CODE REVIEW

**Comment by Aidenkrz** [comment]
`# Goobstation`
:)

File: Resources/Prototypes/Loadouts/loadout_groups.yml [hunk 1]
```diff
@@ -631,13 +631,21 @@
   - StationEngineerPDA
   - SeniorEngineerPDA
 
+- type: loadoutGroup
+  id: AtmosphericTechnicianHead
+  name: loadout-group-atmospheric-technician-head
```
Comment (jorgun) [inline Resources/Prototypes/Loadouts/loadout_groups.yml:636]: add a "minLimit: 0" so we arent foced to have this

**Comment by Aidenkrz** [comment]
erm no

**Comment by BeeRobynn** [comment]
> erm no

7f3cyc-1

**Pull request merged**